web-common/default.nix

```diff
@@ -0,0 +1,11 @@
+{ lib }:
```

Comment:
This one gets a directory-local `default.nix` too!

Reply:
right, so this is happening consistently now.